packages/nextjs/components/onchain-impact-dashboard/projectTotalsGraph/projectTotalsGraph.tsx

```diff
@@ -143,28 +143,34 @@ export const ProjectTotalsGraph = ({
             </div>
 
             <ResponsiveContainer width="100%" className={"absolute top-14"}>
-              <AreaChart data={totalsRecord} margin={{ top: 20, right: -16, bottom: 45, left: 2 }}>
-                <Area
-                  key={metricToWork.name}
-                  type="monotone"
-                  dataKey={metricToWork.name}
-                  stroke={stringToColor(metricToWork.label ?? "")}
-                  fill={stringToColor(metricToWork.label ?? "")}
-                />
-                <CartesianGrid strokeDasharray="3 3" />
-                <XAxis tick={false} fontSize={10} hide={false} dataKey="date" />
-                <YAxis
-                  fontSize={10}
-                  type={"number"}
-                  domain={[minValue - buffer, maxValue + buffer]}
-                  hide={false}
-                  allowDataOverflow={true}
-                  orientation="right"
-                  tickFormatter={v => formatNumber(v, 0)}
-                  ticks={[minValue - buffer, maxValue + buffer]}
-                />
-                <Tooltip content={<CustomTooltip />} />
-              </AreaChart>
+              {isEmpty ? (
+                <div className="flex justify-center items-center w-full h-[300px] text-neutral-content">
+                  No data found yet
```

Comment:
I feel like it would look better to say "no data found" instead of "no data found yet". The latter may make someone feel like it is loading.

Reply:
done